packages/language-service/ivy/completions.ts

```diff
@@ -335,20 +338,36 @@ export class CompletionBuilder<N extends TmplAstNode|AST> {
     }
   }
 
-  private isElementTagCompletion(): this is CompletionBuilder<TmplAstElement> {
-    return this.node instanceof TmplAstElement &&
-        this.nodeContext === CompletionNodeContext.ElementTag;
+  private isElementTagCompletion(): this is CompletionBuilder<TmplAstElement|TmplAstText> {
+    if (this.node instanceof TmplAstText) {
+      const positionInTextNode = this.position - this.node.sourceSpan.start.offset;
+      const textToLeftOfPosition = this.node.value.substr(0, positionInTextNode);
```

Comment:
```suggestion
      const textToLeftOfPosition = this.node.value.substring(0, positionInTextNode);
```

`String.prototype.substr` has been [deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)